src/Controller.js

```diff
@@ -17,17 +17,21 @@ class Controller extends RingaObject {
    * Constructs a new controller.
    *
    * @param id The id of this controller, primarily used for internal hashes and debugging. Must be unique.
-   * @param domNode The native browser DOMNode element (not a React Node) to attach event listeners to.
+   * @param bus The native browser DOMNode element (not a React Node) to attach event listeners to OR a custom event bus.
    * @param options See documentation on Controller options. Defaults are provided, so this is optional.
    */
-  constructor(id, domNode, options) {
+  constructor(id, bus, options) {
     super(id);
 
-    if (__DEV__ && !domNode) {
-      throw Error('Controller:constructor(): no DOMNode provided to constructor!');
+    if (bus && !(typeof bus.addEventListener === 'function')) {
+      if (__DEV__ && options) {
+        throw Error('Controller(): invalid bus or DOM node passed into constructor!');
+      }
+
+      options = bus;
```

Comment:
I don't think lines 26-31 are correct.  It won't throw an error if bus is undefined, but it did if __DEV__ were true and domNode was undefined.